scvi/_settings.py

```diff
@@ -143,12 +144,18 @@ def seed(self) -> int:
         return self._seed
 
     @seed.setter
-    def seed(self, seed: int):
+    def seed(self, seed: Union[int, None] = None):
         """Random seed for torch and numpy."""
-        torch.backends.cudnn.deterministic = True
-        torch.backends.cudnn.benchmark = False
-        seed_everything(seed)
-        self._seed = seed
+        if seed is None:
+            self._seed = None
+            warnings.warn(
+                "Since v1.0.0, scvi-tools will use a random seed by default. Run `scvi.settings.seed = 0` to reproduce results from previous versions."
```

Comment:
```suggestion
                "Since v1.0.0, scvi-tools no longer uses a random seed by default. Run `scvi.settings.seed = 0` to reproduce results from previous versions."
```